webui/react/Makefile

```diff
@@ -47,11 +47,11 @@ check-eslint:
 check-stylelint:
 	npm run stylelint
 check-prettier-js:
-	npm run format-check:js
+	npm run format-check:js > /dev/null
```

Comment:
im not familer with `> /dev/null`. What does that do?

Reply:
i figure it out. How come disabling output of command?

Reply:
`>` only redirect `stdout` which is the happy output of a command. if the command has any warnings or errors to output it will not get redirected by `>`

prettier output one line per file at all times whereas the other commands here don't 

the suggestion here is to limit the output when everything is formatted, as the output doesn't mean much, while not affecting any warnings or errors

Reply:
thanks. I kept the changes out of `npm run ...` commands. if you run those directly you should still get all the output.